zo_haderech.py

```diff
@@ -0,0 +1,19 @@
+#submitted by Keren Or Hadad 208025205
+
+from os import listdir as dirs
+
+# part 5.1
+def zo_haderech(path: str):
```

Comment:
Please add documantation to your function Keren :)

Reply:
done